backend/libs/microservice/src/interfaces/chatbot.ts

```diff
@@ -54,7 +56,7 @@ export function ChatbotServiceControllerMethods() {
         constructor.prototype,
         method,
       );
-      GrpcMethod('ChatbotService', method)(
+      GrpcMethod(CHATBOT_SERVICE_NAME, method)(
```

Comment:
These are generated programatically via `yarn protoc`. I think it will be removed if that command is run

Reply:
Oooo okay